src/python/pants/backend/docker/subsystems/docker_options.py

```diff
@@ -86,16 +92,33 @@ def register_options(cls, register):
             default=[],
             advanced=True,
             help=(
-                "Environment variables to set for `docker` invocations. "
+                "Environment variables to set for `docker` invocations.\n\n"
                 "Entries are either strings in the form `ENV_VAR=value` to set an explicit value; "
                 "or just `ENV_VAR` to copy the value from Pants's own environment."
             ),
         )
 
+        register(
+            "--run-args",
+            type=list,
+            member_type=str,
+            default=[],
+            help=(
+                "Additional arguments to use for `docker run` invocations.\n\n"
```

Comment:
It would be good to give an example:

https://github.com/pantsbuild/pants/blob/763b0c1e5b8380911f1584f8a2168dbcad9198fb/src/python/pants/backend/python/lint/black/subsystem.py#L56-L58